pkg/services/ngalert/models/instance_labels.go

```diff
@@ -100,3 +100,11 @@ func tupleLablesToLabels(tuples tupleLabels) (InstanceLabels, error) {
 	}
 	return labels, nil
 }
+
+func ToMap(lbs InstanceLabels) map[string]string {
```

Comment:
`InstanceLabels` is a `data.Labels` which is a `map[string]string`, so I think you can:

```
func ToMap(lbs InstanceLabels) map[string]string {
	return map[string]string(lbs)
}
```

Also since it is exported, maybe a method instead.

Reply:
Even better, I'll just in-line that the couple places where it's used and remove the method entirely. Thank you!

Reply:
Oh, I recall now why I didn't do this. This actually winds up having a different string representation that where the key for the cache is generated. It winds up `test_uid map[test2:testValue2]` instead of `test_uid test2=testValue2` which is fine but should be internally consistent. I prefer the latter but it's technically no different.